src/commons/asScrollableContent.tsx

```diff
@@ -0,0 +1,82 @@
+import _ from 'lodash';
+import React, {Component} from 'react';
+import {ScrollView, ScrollViewProps} from 'react-native';
+//@ts-ignore
+import hoistStatics from 'hoist-non-react-statics';
+
+interface ScrollableContentProps {
+  scrollableContentProps?: ScrollViewProps;
+  maximumContentSize?: number;
+}
+
+interface ScrollableContentState {
+  maximumContentSize?: number;
```

Comment:
Why should the user need to pass the `maximumContentSize`? 
The whole idea is to measure (using onLayout) the content size vs the container size and do the whole work